amplify/backend/function/expressServer/src/routes/stockRoutes.js

```diff
@@ -18,10 +18,8 @@ router.get('/', protectedRoute, getAllStocks)
 // search stock by ticker symbol. This will give all data except prices for that ticker
 router.get('/:symbol', getStockBySymbol)
 // get the stocks playable in a game
-router.get('/gameStocks/:type', protectedRoute, getGameStocks)
+router.post('/gameStocks/:type', protectedRoute, getGameStocks)
```

Comment:
We are getting game stocks though no why is this post?

Reply:
Its not Amplify man its HTTP, GETS should never have bodys really, searches always use query params take a look at the route GET /api/stocks. We will need to change this anyway man if you want to get working on it while looking at rest of code, post really isn't right here at all I don't think

Reply:
yeah i did a bit of research on it. i could make a big get request with loads of params if needed anyway